app/views/projects/_form.html.erb

```diff
@@ -596,7 +596,25 @@ provided by the delivered project's software.
 </div>
       <br>
       <div class="center">
-      <% if ! is_disabled %>
+      <% if is_disabled %>
+        <% if project.show_entry_license? %>
+          This data is available under the
+          <a href="https://creativecommons.org/licenses/by/3.0/us"
```

Comment:
There is a decent case for CC0. But the longer we have no clear license, the worse.